devtools/dump_devinfo.py

```diff
@@ -46,11 +50,19 @@ def scrub(res):
         "oem_id",
         "nickname",
         "alias",
+        "bssid",
+        "channel",
     ]
 
     for k, v in res.items():
         if isinstance(v, collections.abc.Mapping):
             res[k] = scrub(res.get(k))
+        elif (
+            isinstance(v, list)
+            and len(v) > 0
+            and isinstance(v[0], collections.abc.Mapping)
+        ):
```

Comment:
Would it be enough to just check if it's a list? Might need some restructuring though, and given this is rather a devtool than anything else, I'm fine with it as is.

Reply:
ok cool.